Assets/Scripts/Enemy Folder/IdleState.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Asyncoroutine;
+using UnityEngine.AI;
+using System.Threading;
+
+public class IdleState : MonsterState
+{
+    private float animationDuration;
+    private float animationTimer;
```

Comment:
multiple classes have the same timer and stuff, why not add an interface

Reply:
or put on base class

Reply:
at the moment these are just stand-ins. just need to finalize the stats of the enemy to take there place